bom/application/pom.xml

```diff
@@ -42,7 +42,7 @@
         <microprofile-rest-client.version>2.0</microprofile-rest-client.version>
         <microprofile-jwt.version>1.2</microprofile-jwt.version>
         <smallrye-common.version>1.6.0</smallrye-common.version>
-        <smallrye-config.version>2.3.0</smallrye-config.version>
+        <smallrye-config.version>2.3.1-SNAPSHOT</smallrye-config.version>
```

Comment:
Let's make sure change this :)

Reply:
@geoand My bad I should have added a comment. It's ok @radcortez is aware and [asked me to do it ](https://github.com/smallrye/smallrye-config/pull/579#issuecomment-851676746) 😄

Reply:
Yes, this is so we don't forget the changes required for the next SR Config version.